### ISSUE TITLE
bug from #1128 fixed?

### ISSUE THREAD

**Issue by abfeld007** [nipreps/mriqc#1177]
Hi I am running MRIQC v23.1.0, but I still got the 'dwi' error when I am not using any DWI files (and my dataset passed the bids validator).

```
Traceback (most recent call last):
File "/opt/homebrew/bin/mriqc", line 8, in
sys.exit(main())
^^^^^^
File "/opt/homebrew/lib/python3.11/site-packages/mriqc/cli/run.py", line 206, in main
dataframe, out_tsv = generate_tsv(output_dir, mod)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/opt/homebrew/lib/python3.11/site-packages/mriqc/utils/misc.py", line 181, in generate_tsv
jsonfiles = list(output_dir.glob("sub-/**/%s/sub-_%s.json" % (IMTYPES[mod], mod)))
~~~~~~~^^^^^
KeyError: 'dwi'
```

### What command did you use?

```shell
mriqc NIFTIS output/, 'group'
```

### What version of the software are you running?

23.1.0

### How are you running this software?

Local installation ("bare-metal")

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

No

**Comment by abfeld007** [comment]
Hi, any thoughts?

**Comment by effigies** [comment]
This was fixed in #1148, but there hasn't been a release.

@oesteban @celprov Is there a 24.0 release in the near future, or would it make sense to backport some bug-fixes on top of a `maint/23.1.x` branch and release a 23.1.1?

**Comment by effigies** [comment]
@abfeld007 I assume you `pip install`ed this. You could upgrade to the latest `master` with:

```
pip install git+https://github.com/nipreps/mriqc.git
```

**Comment by oesteban** [comment]
> @oesteban @celprov Is there a 24.0 release in the near future, or would it make sense to backport some bug-fixes on top of a `maint/23.1.x` branch and release a 23.1.1?

we should :) so yes

**Comment by abfeld007** [comment]
yes. I did pip install. I'll try upgrading with the latest master.

**Comment by celprov** [comment]
@effigies @oesteban if you back port some bug fixes and release 23.1.1, you should also merge #1179.

**Comment by NeuroimagingOL** [comment]
Dear all, I also have the same problem when trying to run the group level. I just installed the lastest version, but this problem is still the same. Is there any work around so far or how can I access the group level data?

**Comment by oesteban** [comment]
> This was fixed in #1148, but there hasn't been a release.
> 
> @oesteban @celprov Is there a 24.0 release in the near future, or would it make sense to backport some bug-fixes on top of a `maint/23.1.x` branch and release a 23.1.1?

Last time, I responded too fast without grasping the meaning of the comment. Now, I have just released 23.1.1, started `maint/23.1.x` and will start the roadmap toward releasing a 24.0.0 release.

@abfeld007 @NeuroimagingOL please open a new issue if the problem persists with 23.1.1
